main.js

```diff
@@ -1,94 +1,84 @@
 /* eslint no-unused-vars: 0 no-undef: 0 */
+let id = 0;
 const books = [];
```

Comment:
- [ ] Its okay to use  `arrays`in this milestone but it has to be done inside classes eg 
```
class Books {
  constructor() {
    this.books = [];
  }
  getBooks() {
    return this.books;
  }
}
```
Kindly fix this